cli/src/cli.rs

```diff
@@ -1987,6 +1987,17 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                         .takes_value(true)
                         .required(true)
                         .help("The transaction signature to confirm"),
+                )
+                .after_help(// Formatted specifically for the manually-indented heredoc string
+                   "Note: This will show more detailed information for finalized transactions with verbose mode (-v/--verbose).\
+                  \n\
+                  \nAccount's mode legend in the details:\
+                  \n  |srwx|\
+                  \n    s: this account correctly signed\
+                  \n    r: this account is marked to be read (always true)\
+                  \n    w: this account is marked to be written\
+                  \n    x: this account is marked to be executed via program_id (CPI's program_id can't be marked, though)\
```

Comment:
@t-nelson 

```
$ ./target/release/solana confirm --help
solana-confirm 
Confirm transaction by signature

USAGE:
    solana confirm [FLAGS] [OPTIONS] <TRANSACTION_SIGNATURE>

FLAGS:
    -h, --help                           Prints help information
...

ARGS:
    <TRANSACTION_SIGNATURE>    The transaction signature to confirm

Note: This will show more detailed information for finalized transactions with verbose mode (-v/--verbose).

Account's mode legend in the details:
  |srwx|
    s: this account correctly signed
    r: this account is marked to be read (always true)
    w: this account is marked to be written
    x: this account is marked to be executed via program_id (CPI's program_id can't be marked, though)
```

Reply:
:heart_eyes_cat: Beautiful!